voicer-client/src/components/voice/EditProfile.jsx

```diff
@@ -24,32 +24,36 @@ export default function EditProfile({ profileData, token }) {
   const firstNameInput = useInputControl(initialState.first_name)
   const lastNameInput = useInputControl(initialState.last_name)
   const emailInput = useInputControl(initialState.email)
+  const bioInput = useInputControl(initialState.bio)
 
```

Comment:
maybe try filtering after userInfo = initialState instead of spreading across 4+ lines.